utility/src/main/java/org/oppia/android/util/platformparameter/PlatformParameterConstants.kt

```diff
@@ -62,6 +62,16 @@ annotation class EnableLanguageSelectionUi
 /** Default value for the feature flag corresponding to [EnableLanguageSelectionUi]. */
 const val ENABLE_LANGUAGE_SELECTION_UI_DEFAULT_VALUE = false
 
+/**
+ * Qualifier for the feature flag corresponding to enabling the extra topic tabs - Practice tab
+ * and info tab.
+ */
+@Qualifier
+annotation class EnableExtraTopicTabsUi
+
+/** Default value for the feature flag corresponding to [EnableExtraTopicTabsUi]. */
+const val ENABLE_EXTRA_TOPIC_TABS_UI_DEFAULT_VALUE = false
```

Comment:
Now that this new parameter is added, can we remove the EnablePracticeTab Dagger value?

Reply:
could you please suggest how I should delete this? This has over 1,000 usages in code which are not safe to delete.

Reply:
This module has been mentioned in a lot of component lists (in tests and other places) -- are they to be manually removed? This isn't possible via deletion and refactor.

Reply:
When searching locally for ``EnablePracticeTab`` (with case sensitivity & whole-word matching) I only see 14 matches across 9 files when searching everything. I'm not sure that I'm fully understanding your question since no module changes are needed, I think, unless there are local test modules redefining ``EnablePracticeTab`` (in which case they can just be removed).

Reply:
sorry, it was the PracticeTabModule I had been talking about which has too many usages. Adding this to the agenda,

Reply:
Update: These modules have been completely removed.